doc/sphinxext/sphinx_gallery/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+==============
+Sphinx Gallery
+==============
+
+"""
+import os
+__version__ = '0.1.4'
```

Comment:
Sorry I am confused, I downloaded the latest sphinx from the repository, then should I run the copy script from the nistats doc directory?